### PR TITLE
Remove bintray config and add GH Actions

### LINKED ISSUE #3: Travis CI is dropped
Given the poor handling of the recent security incident (https://arstechnica.com/information-technology/2021/09/travis-ci-flaw-exposed-secrets-for-thousands-of-open-source-projects/) I'm suspending Travis CI integration indefinitely.

Let's move on to GitHub Actions.

### PR DESCRIPTION
**Pull request by hugo-vrijswijk**
Mostly inspired by https://github.com/sbt/sbt-sdlc/

- Remove Bintray references
- Update organization coordinates (I assume this was wanted, but I can change it back)
- Add GH Actions with sbt-ci-release
- Add dependabot for GH Actions
- Add scalafmt
- Ran scalafix for new sbt `/` syntax

Take care while reviewing, it's very possible I missed something. The repo still needs the secrets available in GH Actions before publishing can work

Also closes #3

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '8'
+          cache: 'sbt'
+      - run: sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
```
Comment (hugo-vrijswijk) [inline .github/workflows/release.yml:26]: These will have to be added before releases can work

**Comment by SethTisue** [comment]
I believe @eed3si9n will do the secrets thing

**Comment by eed3si9n** [comment]
Since I'm still active, I prefer my plugins to be in `com.eed3si9n` domain, like https://github.com/sbt/sbt-assembly and https://github.com/sbt/sbt-buildinfo.

**Comment by hugo-vrijswijk** [comment]
Makes sense. I've changed the coords back

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Pull request merged**